### ISSUE TITLE
Open API 中服务查询、示例查询问题

### ISSUE THREAD

**Issue by HankDevelop** [alibaba/nacos#3780]
**Describe the bug**
I access the query service list API, the return value is null
contextPath： nacos/v1/ns/service/list?pageNo=0&pageSize=100
and then I query the instance list API, the return value is null
contextPath： nacos/v1/ns/instance/list?serviceName=teest
Version nacos-server 1.3.1

**Comment by KomachiSion** [comment]
Please provide more information. Such as the params of apis.

**Comment by KomachiSion** [comment]
No

**Comment by KomachiSion** [comment]
From my test, the problem has been fixed in 1.3.2

you can try with demo console 
```
~ » curl -X GET 'console.nacos.io/nacos/v1/ns/service/list?pageNo=0&pageSize=100'                                     
{"doms":[],"count":0}%                                                                                                                                ------------------------------------------------------------------------------------------------------------------------------------------------------
~ » curl -X GET 'console.nacos.io/nacos/v1/ns/service/list?serviceName=teest'                                              
caused: Param 'pageNo' is required.;%                                                                                                                 -------------------------------------------
```
